backend/test/invite_collection_handler_test.py

```diff
@@ -203,77 +196,63 @@ def test_post_invite_user_member_of_other_institution(self, verify_token, send_i
-    def test_post_invite_user_already_member(self, verify_token, send_invite):
-        """ Check if raise exception when the invite is
-        for user already member of institution."""
+    @patch('utils.verify_token', return_value=USER)
+    def test_post_invite_without_admin(self, verify_token, send_invite):
+        """ Check if raise exception when the admin_key is not admistrator."""
         admin = mocks.create_user(ADMIN['email'])
         institution = mocks.create_institution()		 
         admin.institutions_admin = [institution.key]
         institution.admin = admin.key
         admin.add_permission("invite_members",institution.key.urlsafe())
         admin.put()
-        otheruser = mocks.create_user(USER['email'])
-        institution.add_member(otheruser)
         institution.put()
-        body = create_body(USER['email'], admin, institution)
-
+        body = create_body(['ana@gmail.com'], admin, institution)
         with self.assertRaises(Exception) as raises_context:
             self.testapp.post_json("/api/invites", body)
 
         message_exception = self.get_message_exception(str(raises_context.exception))
 
         self.assertEqual(
             message_exception,
-            "Error! The invitee is already a member",
-            "Expected exception message must be equal to "
-            "Error! The invitee is already a member")
+            "Error! User is not allowed to send invites",
+            "Expected exception message must be equal to Error! User is not allowed to send invites")
         
         # assert the invite was not sent
         send_invite.assert_not_called()
-
-    @patch.object(Invite, 'send_invite')
-    @patch('utils.verify_token', return_value=USER)
-    def test_post_invite_without_admin(self, verify_token, send_invite):
-        """ Check if raise exception when the admin_key is not admistrator."""
+    
+    @patch('handlers.invite_collection_handler.enqueue_task')
+    @patch('utils.verify_token', return_value=ADMIN)
+    def test_post_many_invites_at_once(self, verify_token, enqueue_task):
         admin = mocks.create_user(ADMIN['email'])
-        institution = mocks.create_institution()		 
+        institution = mocks.create_institution()
         admin.institutions_admin = [institution.key]
+        admin.add_institution(institution.key)
         institution.admin = admin.key
-        admin.add_permission("invite_members",institution.key.urlsafe())
+        admin.add_permission("invite_members", institution.key.urlsafe())
         admin.put()
         institution.put()
-        body = create_body('ana@gmail.com', admin, institution)
-        with self.assertRaises(Exception) as raises_context:
-            self.testapp.post_json("/api/invites", body)
-
-        message_exception = self.get_message_exception(str(raises_context.exception))
+        body = create_body(
+            ['ana@gmail.com', 'user@hotmail.com', 'test@example.com', 'other@other.com'], 
```

Comment:
You shouldn't use "real" names in the tests, nor real email domains.

Reply:
It hasn't been me